### PR TITLE
Adds custom pipe and usage

### PR DESCRIPTION
**Pull request by mjrdnk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -366,6 +366,35 @@ angular.module('myReverseFilterApp', [])
...
 
+### Angular
+
+In Angular filters are called pipes.
+Built-in pipes available in Angular: DatePipe, UpperCasePipe, LowerCasePipe, CurrencyPipe, and PercentPipe.
+More about pipes in Angular [here](https://angular.io/guide/pipes)
+
+Apart from built in, you can create your own, custom pipes.
+
+Create custom pipe:
```
Comment (feimosi) [inline README.md:410]: Could you briefly mention what this pipe does?

**Review by feimosi — CHANGES_REQUESTED** [review]
Cool, thanks!

**Comment by mjrdnk** [comment]
Hey, I updated the PR with some explanatory text.

**Comment by feimosi** [comment]
Perfect! Could you resolve the merge conflicts?

**Comment by mjrdnk** [comment]
Done

**Pull request merged**